client/src/pages/Login.js

```diff
@@ -0,0 +1,193 @@
+        color: '#fff',
+        fontWeight: 300,
+        textTransform: 'none',
+        margin: '40px auto',
+        padding: '8px 30px',
+        '&:hover': {
+            backgroundColor: 'darkorange'
+        }
+    },
+
+    textFooter: {
+        fontWeight: 500
+    },
+
+}));
+
+function Login() {
+    const classes = useStyles();
+    const [confirmed, setConfirmed] = React.useState(false);
+    const [email, setEmail] = React.useState('');
+
+    const handleChange = (e) => {
+        setEmail(e.target.value)
+    }
+
+    const handleGetStarted = () => {
+        setConfirmed(true)
+    }
+
+    const responseGoogle = (response) => {
+        let status;
+        fetch("/googlelogin", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify({ tokenId: response.tokenId })
+        })
+            .then(res => {
+                status = res.status;
+                if (status < 500) return res.json();
+                else throw Error("Server error");
+            })
+            .then(res => {
+                if (status === 200) alert(res.response);
+                const cookies = new Cookies();
+                cookies.set('token', res.token, { path: '/', httpOnly: true })
+                if (status === 401) alert(res.response);
+            })
+            .catch(err => {
+                alert(err.message);
+            });
+    }
+
+
+    return (
+        <Grid container direction="column" alignItems="center">
+            <Grid item>
+                <Link href="/">
+                    <img src={Logo} alt="calendapp" className={classes.logo} />
+                </Link>
+            </Grid>
+
+            <Grid item className={classes.shadowCard}>
+                {confirmed === false && (
+                    <CardContent className={classes.body}>
+                        <Typography variant="h6" className={classes.title}>
+                            Log into your account
+                            </Typography>
+                        <Typography className={classes.label}>
+                            Enter your e-mail to get started:
+                            </Typography>
+                        <BootstrapInput fullWidth id="email" placeholder="E-mail address"
+                            type="email" variant="outlined" className={classes.input}
+                            onChange={handleChange} value={email} />
+                        <Button className={classes.btnOrange}
+                            onClick={handleGetStarted}>Continue</Button>
+                    </CardContent>
+                )}
+                {confirmed === true && (
+                    <CardContent className={classes.body}>
+                        <Typography variant="h6" className={classes.title}>
+                            Welcome back,
+                                <br />{email}
+                        </Typography>
```

Comment:
Small thing, but you could probably split these two lines into separate `<Typography>` components, and you wouldn't have to use the `<br/>`.